main.tex

```diff
@@ -18,6 +18,49 @@
+In der Realität besteht die Nutzerbasis wie besagt aus männlichen Jugendlichen und jungen Erwachsenen. Hier wurde Facebook zum Teil ein integrierter Teil des Lebens, das sich fast virusartig durch persönliche Weiterempfehlung verbreitete. Dies geht über zu spezialisierten Kursen von neuen oder schon vorher "Tech-Affinen", die Teilnehmern Grundlegendes von der Inbetriebnahme von Smartphones bis zum Kommentieren bei Facebook beibringen. \cite{empowermentThroughFacebook}
+
+\medskip
+
+Diese Verbreitung zog selbstverständlich eine Verschiebung der sozialen Normen nach sich, besonders bei Fragen der sozialen Akzeptanz unter den Jugendlichen. Der Besitz eines "guten", sprich aktuellen Smartphones und viele (auch internationale) Freunde auf Facebook wurden Teil des neuen Selbst, ähnlich wie es schon im Westen der Fall war. Es folgte eine Mentalität des ewigen Kreislaufs des Neids; "I want what he has". \cite{empowermentThroughFacebook}
+
+\medskip
+
+ Ein weiterer Aspekt ist die Misrepräsentation der eigenen Person 'online', was immer mehr zur Norm wurde. So wird ein Uniabbruch plötzlich zum bestandenen Bachelor, ein solcher zum sogennanten "Honours degree", eine "höher gestellte Variante". Ursprung dieser Entwicklung sei ein allgemeines Gefühl der Nichtdazugehörigkeit, die Generation der gelehrten, arbeitslosen zwanzig- bis dreißigjährigen Indern, eine Generation, die von der Welt "abgehängt" wurde. Das Lehren des Internets an Andere wird hier zum Daseinszweck, gleiches gilt für die Verbindung zu internationalen Freunden via 'Facebook' und 'Google Translate'. \cite{empowermentThroughFacebook}
+
+\medskip
+
+Besagte Kurse brachten einen weiteren positiven Effekt mit sich, die Verbreitung des Englischen durch Selbstbeibringung in Gebieten, in denen vorher weder Englisch noch Hindi gesprochen wurde. \cite{empowermentThroughFacebook} Die Tatsache, dass der Großteil der Analphabeten vor Allem unter der genannten Gruppe, welche unter der Armutsgrenze lebt, zu finden sind, bleibt hierbei leider bestehen.
+
+\medskip
+
+Ferner stehen positiven Aspekten weitere bereits etablierte soziale Normen im Wege; Informationsverbreitung geschieht im ländlichen Indien immer noch zu großen Teilen über Mundpropaganda \cite[259]{everydayLife}, Mädchen wird Zugriff auf Medien zugunsten des (erstgeborenen) Jungen verwährt \cite{empowermentThroughFacebook}, soziale Gruppierungen werden vom Realen ins Virtuelle übernommen.
+
+\medskip
+
+All dies hätte sich nicht durch eine kostenpflichtige Variante des Internets geändert. Allein die Zahlen der nationalen Internetverbreitungsrate (2016 25\% \--> 2019 50\%) sprechen für eine graduelle Veränderung gen Akzeptanz bezüglich des Internets. %I think this should be left out - or changed significantly - need opinions
```

Comment:
> Kostenpflichtige

kostenlose* 

>%I think this should be left out - or changed significantly - need opinions

Ich weiß, was du meinst. Die Frage ist ob die Verbreitungsrate sich auch signifikant in ländlichen Gebieten erhöht, oder lediglich hauptsächlich in den Städten. Muss ich mir nochmal angucken, ggf kann ich hier nochmal was adden, was mir so durch den Kopf geht